quinn-proto/src/cid_queue.rs

```diff
@@ -15,7 +15,7 @@ pub struct CidQueue {
     ///
     /// The CID sequenced immediately prior to this is the active CID, which this data structure is
     /// not responsible for retiring.
-    offset: u64,
+    pub(crate) offset: u64,
```

Comment:
Rather than exposing this directly to external manipulation, can we encapsulate the operations we're trying to perform on it? It's much easier to reason about the possible states of a data structure if its innards aren't exposed to large amounts of code.

Reply:
Sure. Will encapsulate with some functions on this field